src/Core/Package.vala

```diff
@@ -379,7 +379,6 @@ public class AppCenterCore.Package : Object {
         color_primary_text = null;
         payments_key = null;
         suggested_amount = null;
-        _latest_version = null;
```

Comment:
This methods resets all of the cached data on the package when we reload the AppStream data. Since the latest version string doesn't come from AppData and instead comes from the deb package, we don't need to drop this cache.